src/components/common/validation/time-format/ValidatedTimeEntry.jsx

```diff
@@ -8,7 +8,22 @@ const ValidatedTimeEntry = ({
   register,
   isRequired,
 }) => {
-  const errorMessage = `You must enter a ${timeType} in the HH:MM 24 hour clock format`;
+  const validateTime = (time) => {
```

Comment:
nit: ever so slightly misleading function name **given** the return value. It suggests the boolean represents if the time was validated or not. 
e.g. isTimeValid / timeValid would be a bit more explicit.
just a suggestion but feel free to just resolve this.

Reply:
I've renamed this to a clearer name